### ISSUE TITLE
python3.7 compatibility enum34 error

### ISSUE THREAD

**Issue by creisle** [biocommons/hgvs#584]
package version: 1.3.0.post0
python version: 3.7.2
os version: centos07

When I install hgvs I have the following issue upon running any code

```python
  File "/gsc/software/linux-x86_64-centos7/python-3.7.2/lib/python3.7/re.py", line 143, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'
```

I tracked this down and from this SO question https://stackoverflow.com/questions/43124775/why-python-3-6-1-throws-attributeerror-module-enum-has-no-attribute-intflag I was able to determine enum34 as the culprit.

I looked through the dependency tree and the only package I am using with enum34 as a dependency is this one. Is there a way to not have it a requirement to avoid breaking 3.7+? or an alternate solution?

**Comment by reece** [comment]
I agree that it appears that enum34 can be removed. 
I'll need at least a few days to get a new release out.

**Comment by creisle** [comment]
Great! thanks for looking into it

**Comment by reece** [comment]
@creisle This should be fixed in hgvs 1.4.0, available from github and on pypi.

I would appreciate it if you'd test and report back back here. If fixed, please close the issue. Thanks!

**Comment by reece** [comment]
Closing. Reopen if problem persists.
